app/views/devise/unlocks/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend unlock instructions</h2>
```

Comment:
Hmmm... your user model is not `lockable`, so don't include this form.
(I know it was generated by devise, but don't include files you're not using).